dsf-fhir/dsf-fhir-server/src/main/java/org/highmed/dsf/fhir/dao/provider/DaoProviderImpl.java

```diff
@@ -359,7 +372,7 @@ public ValueSetDao getValueSetDao()
 	public <R extends Resource> Optional<? extends ResourceDao<R>> getDao(Class<R> resourceClass)
 	{
 		@SuppressWarnings("unchecked")
-		ResourceDao<R> value = (ResourceDao<R>) daosByResourecClass.get(resourceClass);
+		ResourceDao<R> value = (ResourceDao<R>) daosByResoureClass.get(resourceClass);
```

Comment:
Typo: change to `daosByResourceClass`?

Reply:
fixed via [ecad59f](https://github.com/highmed/highmed-dsf/pull/333/commits/ecad59fcfeecc95b1b25bf89c5ccc7a5066fbed0)